Bloom.md

```diff
@@ -0,0 +1,341 @@
+---
+RFC: 4
+Status: Proposed
+---
+
+# ValkeyBloom Module RFC
+
+## Abstract
+
+The proposed feature is ValkeyBloom which is a Rust based Module that brings a native bloom filter data type into Valkey.
+The module should be API-compatible with Redis Ltd.’s ReBloom Module.
+
+## Motivation
+
+Bloom filters are a space efficient probabilistic data structure that can be used to “check” whether an element exists in
```

Comment:
```suggestion
Bloom filters are a space efficient probabilistic data structure that can be used to "check" whether an element exists in
```
I just realized these are curly quotes. Probably fine to use them, but it would make sense to keep this all as ascii.

Reply:
> I just realized these are curly quotes. Probably fine to use them, but it would make sense to keep this all as ascii.

Makes sense. This has been removed